wiki/Ranking_Criteria/Scaling_BPM/id.md

```diff
@@ -0,0 +1,25 @@
+# Penskalaan BPM di kriteria ranking
+
+Aturan dan pedoman Kriteria Ranking didasarkan pada lagu-lagu dengan 180 BPM dengan tanda birama 4/4. Panduan ini menjelaskan cara kerja penskalaan itu!
+
+Kami akan menggunakan panduan untuk tingkat kesulitan Normal osu! sebagai contoh:
+
+> **Sertakan jeda ritme 1/1 atau lebih panjang untuk setiap dua garis birama gameplay.**
+
+Untuk kesederhanaan, pikirkan 180 BPM sebagai titik tengah.
+
+- **Saat tempo lagu berubah dari 180 BPM menjadi ~120 BPM,** pedoman menjadi lebih lembut. Untuk contoh ini, peningkatan frekuensi ritme 1/2 pada umumnya tidak masalah hingga rantaian 1/2 hampir konstan pada 120 BPM.
+- **Saat tempo lagu turun di bawah ~120 BPM,** ritme 1/4 dapat diperkenalkan. Semakin dekat dengan 120 BPM, berarti semakin langka ritme 1/4 dipakai dalam rantai yang terdiri dari 2 objek atau lebih.
```

Comment:
I think `dapat diperkenalkan` can be changed to `mulai muncul` or `mulai terlihat`. The use of `diperkenalkan` in Indonesian seems to indicate that the rhythm is introducing itself.

For `berarti semakin langka ritme 1/4`, i suggest to change `langka` to `sedikit`, using the same word as below: `ritme 1/2 yang sedikit dapat diterima.`